packages/jobs/lib/runner/local.runner.ts

```diff
@@ -51,7 +51,7 @@ export class LocalRunner implements Runner {
                 env: {
                     ...process.env,
                     RUNNER_ID: runnerId,
-                    IDLE_MAX_DURATION_MS: '60000'
+                    IDLE_MAX_DURATION_MS: '0'
```

Comment:
Unsure why we were idling on the local runner @TBonnin? Maybe as part of testing?

Reply:
because it mimics how it actually work in production and since starting a runner locally is super fast I didn't see any harm having the idling functionality enabled. Did it cause any issue for you?

Reply:
No issue, just was very noisy when I was trying to look at the logs on my local.

Reply:
It might be useful to detect if the feature stops working but I understand it adds some noise. Your call, I don't mind either way